requirements/edx/github.txt

```diff
@@ -101,6 +101,7 @@ git+https://github.com/edx/xblock-utils.git@v1.0.5#egg=xblock-utils==1.0.5
 git+https://github.com/edx/edx-user-state-client.git@1.0.2#egg=edx-user-state-client==1.0.2
 git+https://github.com/edx/xblock-lti-consumer.git@v1.1.6#egg=lti_consumer-xblock==1.1.6
 git+https://github.com/edx/edx-proctoring.git@1.3.1#egg=edx-proctoring==1.3.1
+-e git+https://github.com/edx/edx-val.git@mrehan/download-transcripts-on-upload-page#egg=edxval
```

Comment:
Make sure this is removed from here and appropriate VAL version is bumped into base requirements.